Document/0x04e-Testing-Authentication-and-Session-Management.md

```diff
@@ -561,12 +561,22 @@ For additional best practices and detailed information please refer to the follo
 
 ### Testing Login Activity and Device Blocking (MSTG-AUTH-11)
 
-For applications which require L2 protection, the MASVS states that: "The app informs the user of all login activities with their account. Users are able view a list of devices used to access the account, and to block specific devices.". This can be broken down into various scenarios:
+For applications which require L2 protection, the MASVS states that: "The app informs the user of all login activities with their account. Users are able view a list of devices used to access the account, and to block or allow specific devices.". 
 
-1. The application provides a push notification the moment their account is used on another device to notify the user of different activities. The user can then block this device after opening the app via the push-notification.
-2. The application provides an overview of the last session after login, if the previous session was with a different configuration (e.g. location, device, app-version) then the user his current configuration. The user then has the option to report suspicious activities and block devices used in the previous session.
-3. The application provides an overview of the last session after login at all times.
-4. The application has a self-service portal in which the user can see an audit-log and manage the different devices with which he can login.
```

Comment:
removing these 4 items might not be a good idea.

Reply:
the thing was that the items were re-explained diffrently and i tried to add more detailed. but we can keep them i just wanted to avoid redundancy .

Reply:
I'd go for a merge of both things, as both have good content :)